rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/internal/LRARecoveryModule.java

```diff
@@ -200,53 +241,66 @@ public void getRecoveringLRAs(Map<URI, LongRunningAction> lras) {
      * @return false if record isn't in the store or there was an error removing it
      */
     public boolean removeCommitted(Uid lraUid) {
-        try {
-            return _recoveryStore.remove_committed(lraUid, _transactionType);
-        } catch (ObjectStoreException e) {
-            if (LRALogger.logger.isTraceEnabled()) {
-                LRALogger.logger.tracef(e,
-                        "LRARecoveryModule: Object store exception '%s' while removing LRA record %s",
-                        e.getMessage(), lraUid.fileStringForm());
-            } else if (LRALogger.logger.isInfoEnabled()) {
-                LRALogger.logger.infof(
-                        "LRARecoveryModule: Object store exception '%s' while removing LRA record %s",
-                        e.getMessage(), lraUid.fileStringForm());
+        synchronized (this) {
```

Comment:
synchronized here, is necessary?

Reply:
I will remove it for same reason as discussed earlier.